packages/connect/src/types/api/bitcoin/index.ts

```diff
@@ -2,17 +2,19 @@ import type { AccountAddresses } from '@trezor/blockchain-link';
 import type { Transaction as BlockbookTransaction } from '@trezor/blockchain-link-types/lib/blockbook';
 import type { PROTO } from '../../../constants';
 import type { AccountTransaction } from '../../account';
-import type { DerivationPath, ProtoWithDerivationPath } from '../../params';
+import { DerivationPath, ProtoWithDerivationPath } from '../../params';
+import { Static, Type } from '@trezor/schema-utils';
 
 // signMessage
 
-export interface SignMessage {
-    path: DerivationPath;
-    coin: string;
-    message: string;
-    hex?: boolean;
-    no_script_type?: boolean;
-}
+export type SignMessage = Static<typeof SignMessage>;
+export const SignMessage = Type.Object({
+    path: DerivationPath,
+    coin: Type.Optional(Type.String()),
+    message: Type.String(),
+    hex: Type.Optional(Type.Boolean()),
+    no_script_type: Type.Optional(Type.Boolean()),
+});
 
 // signTransaction
```

Comment:
there is a failed type-test https://github.com/trezor/trezor-suite/actions/runs/7387348432/job/20095940761?pr=10280#step:15:412
but signTransaction wasn't really touched (that much). 🤔